extensions/dicom-seg/src/components/SegmentationSelect.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Select from 'react-select';
+
+const SegmentationSelect = ({ value, formatOptionLabel, options }) => (
+  <Select
+    value={value}
+    formatOptionLabel={formatOptionLabel}
+    styles={segmentationSelectStyles}
+    options={options}
+  />
+);
+
+const segmentationSelectStyles = {
+  control: (base, state) => ({
+    ...base,
+    cursor: 'pointer',
+    background: '#151A1F',
```

Comment:
These colors should be imported from css style vars.

Reply:
Done, thanks.